AnkiDroid/src/main/java/com/ichi2/utils/LanguageUtil.kt

```diff
@@ -0,0 +1,111 @@
+/*
+ Copyright (c) 2020 David Allison <davidallisongithub@gmail.com>
+
+ This program is free software; you can redistribute it and/or modify it under
+ the terms of the GNU General Public License as published by the Free Software
+ Foundation; either version 3 of the License, or (at your option) any later
+ version.
+
+ This program is distributed in the hope that it will be useful, but WITHOUT ANY
+ WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License along with
+ this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+package com.ichi2.utils
+
+import android.content.SharedPreferences
+import android.content.res.Resources
+import android.text.TextUtils
+import androidx.core.os.ConfigurationCompat
+import com.ichi2.anki.AnkiDroidApp
+import com.ichi2.anki.Preferences
+import timber.log.Timber
+import java.text.DateFormat
+import java.util.*
+
+/**
+ * Utility call for proving language related functionality.
+ */
+object LanguageUtil {
+    /** A list of all languages supported by AnkiDroid
+     * Please modify LanguageUtilsTest if changing
+     * Please note 'yue' is special, it is 'yu' on crowdin, and mapped in import specially to 'yue'  */
```

Comment:
```suggestion
     * Please note 'yue' is special, it is 'yu' on Crowdin, and mapped in import specially to 'yue'  */
```

The conflict it took is just the typo change from here https://github.com/ankidroid/Anki-Android/commit/5da9561a290122b52047c5c24648f078dc02dca7#diff-4ab81775efd50ce812d06d139972eb25292efd73500eb86efdb15027ad453779 - easy to ingest in the migration and mark as de-conflicted